### PR TITLE
my first PR

### PR DESCRIPTION
**Pull request by emiliapirvanus**
this is my PR (pull request)

### CODE REVIEW

File: src/main/java/com/learning/petstore/controller/PetController.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+public class PetController {
```
Comment (robertpirvanus) [inline src/main/java/com/learning/petstore/controller/PetController.java:18]: i do not like this

File: src/main/java/com/learning/petstore/controller/PetController.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    @Autowired
```
Comment (robertpirvanus) [inline src/main/java/com/learning/petstore/controller/PetController.java:19]: neither this

File: src/main/java/com/learning/petstore/controller/PetController.java (hunk 1 above)
Comment (emiliapirvanus) [reply src/main/java/com/learning/petstore/controller/PetController.java:18]: i will fix it

File: src/main/java/com/learning/petstore/repository/entities/Pet.java [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+import javax.persistence.*;
+import java.util.List;
+import java.util.Optional;
+
+@Entity
+@Table(name = "pet")
+public class Pet {
+    @Id
+    @Column(name = "id")
+//    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "pet_seq")
```
Comment (robertpirvanus) [inline src/main/java/com/learning/petstore/repository/entities/Pet.java:12]: you should not leave code comments in the code

**Review by robertpirvanus — APPROVED** [review]
good. i will approve

**Pull request merged**